drivers/adc/Kconfig.vf610

```diff
@@ -0,0 +1,47 @@
+# ADC configuration options
+
+# Copyright (c) 2021 Antonio Tessarolo
+# SPDX-License-Identifier: Apache-2.0
+
+config ADC_VF610
+	bool "VF610 ADC driver"
+	depends on HAS_VF610_ADC
+	help
+	  Enable the VF610 ADC driver.
+
+if ADC_VF610
+
+choice
+	prompt "Clock Source Select"
+	default ADC_VF610_CLK_IPG_DIV_2
+
```

Comment:
This can move to the device tree. See the below for example.
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/adc/nxp%2Ckinetis-adc12.yaml#L17-L25